conda/recipes/libcuspatial/meta.yaml

```diff
@@ -34,18 +34,35 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
     - {{ compiler('cxx') }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }} {{ cuda_version }}
+    - cuda-cudart ={{ cuda_version }}
+    - cuda-cudart-dev ={{ cuda_version }}
+    - cuda-cudart-static ={{ cuda_version }}
+    - cuda-cupti ={{ cuda_version }}
+    - cuda-driver-dev {{ cuda_driver_version }}
+    {% endif %}
     - cmake {{ cmake_version }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    {% else %}
+    - cuda-cudart ={{ cuda_version }}
+    - cuda-cudart-dev ={{ cuda_version }}
+    - cuda-cudart-static ={{ cuda_version }}
+    - cuda-cupti ={{ cuda_version }}
+    - cuda-driver-dev {{ cuda_driver_version }}
+    {% endif %}
     - doxygen
     - gmock {{ gtest_version }}
     - gtest {{ gtest_version }}
-    - libcudf ={{ minor_version }}
-    - librmm ={{ minor_version }}
+    - libcudf ={{ minor_version }}*=cuda{{ cuda_major }}*
```

Comment:
Revert these. We won’t pin on build strings, we are using the cuda-version dependency to constrain which packages work together.